package.json

```diff
@@ -9,9 +9,10 @@
     "compile": "webpack",
     "karma": "karma start karma.config.js",
     "lint": "eslint '**/*.js'",
+    "lint-fix": "eslint '**/*.js' --fix",
     "prepublish": "yarn compile",
     "start": "webpack-dev-server",
-    "test": "yarn ci:lint && yarn ci:test",
+    "test": "yarn ci:test && yarn ci:lint",
```

Comment:
Yeah, otherwise pushing unlinted code will just result in more unlinted code and then complaints about code not being linted and failing CI builds.

Please switch it back.